meta-flex-distro/recipes-core/packagegroups/packagegroup-core-tools-profile.bbappend

```diff
@@ -4,3 +4,6 @@
 
 # Flex OS Flex does not support systemtap. Systemtap brings boost which takes lots of resources. So we do not need it.
 SYSTEMTAP:flex-os = ""
+
+# Add perfetto to tools-profile for tracing
+RDEPENDS:${PN}:append:flex-os = " perfetto"
```

Comment:
Shouldn't we add a check for tracing user/distro feature because otherwise it's unconditionally changing the behavior of an upstream packagegroup.

Reply:
well, it is debatable. From other viewpoint it is saying our distro define core tools profile like this. This recipe will only be included in our image when tracing is enabled. So current solution looks fine to me.

Reply:
OK, let's merge it then.